modules/ROOT/pages/connecting-salesforce-orgs.adoc

```diff
@@ -0,0 +1,80 @@
+= Establish a Tenant Relationship With a Trusted Salesforce Organization
+
+Establishing a tenant relationship between an Anypoint Platform organization and a Salesforce organization enables users in your organization to:
+
+* Publish invocable actions directly from their Robotic Process Automations (RPA) and Composer flows.
+* Enhance Anypoint Code Builder and Intelligent Document Processing (IDP) developer experiences with Einstein. 
+* Make MuleSoft APIs available in the connected Salesforce organization's API Catalog.
+
+When you establish the tenant relationship, you can initiate, complete, and manage the tenant relationships. You can also enable, disable, and remove connections. 
+
+Tenant relationships are initiated in Anypoint Platform.   
+
+These are the ways a tenant relationship is established:
+
+* A Salesforce organization is provisioned with the tenant relationship to an Anypoint Platform organization already established. In this case, the Anypoint Platform organization administrator just needs to xref:managing-connected-salesforce-orgs.adoc#enable-disable-connection[enable the connection] in Anypoint Platform. 
+* The tenant relationship is created by the Anypoint Platform organization administrator in Anypoint Platform and setup is completed in the Salesforce organization by the Salesforce organization administrator.  
```

Comment:
The Anypoint Platform organization administrator in Anypoint Platform creates the tenant relationship and the Salesforce org admin completes the setup in the Salesforce org.